packages/nitro/src/runtime/workflow-runtime-config.ts

```diff
@@ -0,0 +1,55 @@
+import { defineNitroPlugin } from 'nitro/~internal/runtime/plugin';
+import { useRuntimeConfig } from 'nitro/runtime-config';
+
+/**
+ * Bridge Nitro runtimeConfig -> Workflow local world env vars.
+ *
+ * This is framework-agnostic:
+ * - Works in Nuxt because Nuxt feeds runtimeConfig into Nitro.
+ * - Works in standalone Nitro if the user provides `runtimeConfig` in nitro.config.
+ *
+ * Supported keys:
+ * - `runtimeConfig.workflow.baseUrl` (preferred, private/server-only)
+ * - `runtimeConfig.public.workflow.baseUrl` (fallback)
+ * - `runtimeConfig.workflow.{protocol,host,port}` (preferred)
+ * - `runtimeConfig.public.workflow.{protocol,host,port}` (fallback)
+ *
+ * Result:
+ * - Sets `process.env.WORKFLOW_LOCAL_BASE_URL` if not already set.
+ */
+export default defineNitroPlugin(() => {
+  if (process.env.WORKFLOW_LOCAL_BASE_URL) return;
+
+  const runtimeConfig = useRuntimeConfig() as any;
```

Comment:
i don't think we need the `any` keyword here right? TS doesn't seem to complain

```suggestion
  const runtimeConfig = useRuntimeConfig();
```